// Message proto to push downlink traffic in mqtt server (content of topic v3/app1@tenant1/devices/dev1/down/push)
syntax = "proto2";

package iot;

message Downlink {
  optional int32 f_port = 1;
  optional string frm_payload = 2;
  optional string priority = 3;
}

message Downlinks {
  repeated Downlink downlink = 1;
}